integration-tests/utils/integration-helpers.js

```diff
@@ -211,6 +251,40 @@ const withUser = (test) => async (args) => {
   }
 };
 
+/** Deletes all v2 apps in a workspace. Returns true if all deletes succeed. */
+const deleteAppsV2 = async ({ page, billingProject, workspaceName }) => {
+  const workspaceId = await page.evaluate(
```

Comment:
Put this in a helper function/const since you call it from the deleteRuntimesV2 function also.